core/src/main/java/com/netflix/conductor/core/execution/tasks/SubWorkflow.java

```diff
@@ -56,6 +58,16 @@ public void start(WorkflowModel workflow, TaskModel task, WorkflowExecutor workf
                     objectMapper.convertValue(
                             input.get("subWorkflowDefinition"), WorkflowDef.class);
             name = workflowDefinition.getName();
+        } else {
+            name =
+                    Optional.ofNullable(input.get("subWorkflowName"))
+                            .map(Object::toString)
+                            .orElse(null);
+            version = (Integer) input.get("subWorkflowVersion");
```

Comment:
Should we default to latest version in case it is missing from the input?

Reply:
this is done in taskmapper